### ISSUE TITLE
If you change your in-game main hand to the left you can still only do stuff with your right

### ISSUE THREAD

**Issue by Ch1216** [vini2003/Body-Shuffle#5]
If you change your in-game main hand to the left you can still only do stuff with your right
(You can change your main hand by going to options > Skin Customization and there you can change your Main Hand)

**Comment by ohnecksThing** [comment]
Yeah, found that out as well, mod doesn't count your main hand, it always counts right as main one
